simple/src/simple/ScannerAllDataType.java

```diff
@@ -0,0 +1,41 @@
+package simple;
+
+import java.util.Scanner;
+
+public class ScannerAllDataType {
+
+	public static void main(String[] args) {
+
+		byte b;
+		System.out.println("Enter byte");
+		Scanner sc = new Scanner(System.in);
```

Comment:
remove scanner

Reply:
create other program for without scanner